src/linear_algebra/free_module_pid.lean

```diff
@@ -0,0 +1,430 @@
+/-
+Copyright (c) 2020 Anne Baanen. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Anne Baanen
+-/
```

Comment:
Git seems to be having a hard time tracking the diff of this file.

Since bors will squash all the commits together, can you make a PR that only renames this file which can be merged first?

Reply:
Sure, no problem.

Reply:
#7805